docs/setup/local.md

```diff
@@ -8,32 +8,71 @@ Deploying KubeEdge locally is used to test, never use this way in production env
 
 ## Setup Cloud Side (KubeEdge Master Node)
 
+### Create CDRs
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha1_device.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/devices/devices_v1alpha1_devicemodel.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/reliablesyncs/cluster_objectsync_v1alpha1.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubeedge/kubeedge/master/build/crds/reliablesyncs/objectsync_v1alpha1.yaml
+```
+
+
 ### Prepare config file
 
 ```shell
 # cloudcore --minconfig > cloudcore.yaml
 ```
 
-Update any fields if needed.
+Update any fields if needed, details refer to [this](../configuration/kubeedge.md#configuration-edge-side-kubeedge-worker-node).
```

Comment:
change to something like this,
```
please refer to [configuration for edge](...) for details
```

and link should be linked to cloud configuration instead of edge.